tests/fixture_tailwind_remote_classes/nestedPlugins/nested/nested/nested/veryNestedPlugin.ts

```diff
@@ -0,0 +1,16 @@
+import type { Plugin } from "../../../../../../src/server/types.ts";
+import VeryNestedPluginComponent from "../../../../components/VeryNestedPluginComponent.tsx";
+
+const currentUrl = new URL(import.meta.url);
+currentUrl.pathname = currentUrl.pathname.split("/").slice(0, -5).join("/") +
+  "/";
+
+export const veryNestedPlugin = {
+  name: "very nested plugin",
+  location: import.meta.url,
+  projectLocation: currentUrl.href,
```

Comment:
```suggestion
import { toFileUrl } from "$std/path/to_file_url.ts";

export const veryNestedPlugin = {
  name: "very nested plugin",
  location: import.meta.url,
  projectLocation: toFileUrl(Deno.cwd()).href,
```
Ditto for `tests/fixture_tailwind_remote_classes/nestedPlugins/nestedPlugin.ts`.

Reply:
I'm not sure about this...

What if I do `deno run -A tests/fixture_tailwind_remote_classes/dev.ts`? I don't think `Deno.cwd()` would give the right answer then.

Reply:
This is true. My suggestion would only work for the Fresh test that spawns a sub-process in the `tests/fixture_tailwind_remote_classes` folder. Is there a cleaner alternative that lets the reader know what `currentUrl.href` evaluates to? It's not so clear right now.

Reply:
It seems like there should be an easy way of going up N levels, based on starting with `import.meta.url`.

Reply:
Something like this
```ts
export function upNLevels(url: string | URL, levels: number) {
  const inputUrl = (typeof url === "string") ? new URL(url) : url;

  const segments = inputUrl.pathname.split(path.SEP);

  if (levels >= segments.length) {
    throw new Error("Cannot go up more levels than the current depth");
  }

  const upperPathSegments = segments.slice(0, -levels);
  const newPath = path.join(...upperPathSegments);

  inputUrl.pathname = newPath;

  return inputUrl.href;
}
```

Then we can simplify the plugin declaration like this:
```ts
export const nestedPlugin = {
  name: "nested plugin",
  location: import.meta.url,
  projectLocation: upNLevels(import.meta.url, 2),
  routes: [{
    path: "routeFromNestedPlugin",
    component: NestedPluginComponent,
  }, {
    path: "atRouteFromNestedPlugin",
    component: AtPluginComponent,
  }],
} satisfies Plugin;
```

This approach avoids any issue with file vs http urls and relying on `Deno.cwd`.

Reply:
Also let me know if this one could live in `deno_std`.